requirements/prod-requirements.txt

```diff
@@ -105,6 +106,7 @@ markdown==2.2.1
 markupsafe==1.0
 mock==2.0.0
 monotonic==1.5            # via eventlet
+msgpack-python==0.5.6     # via ddtrace
```

Comment:
`msgpack-python` is in uninstall requirements

Reply:
Thanks for catching that, fixed here https://github.com/dimagi/commcare-hq/pull/22067/commits/2916850d60d1846f5b3331f0f948331bdf34b903